shared/tests/utils/sms.test.ts

```diff
@@ -0,0 +1,29 @@
+import { TWILIO_SID, TWILIO_TOKEN } from '../../src/config';
+import { sendSms } from '../../src/utils/messaging/sms';
+
+describe('send simple Sms', () => {
```

Comment:
I guess this will not work in the github actions without having the proper env config there?

Reply:
Yeah correct, but is this a problme?

Reply:
No, I think that's fine 👍 .

I think all tests should pass when one runs them locally after cloning a repo.
I would suggest that we can run this test when env is configured with the proper credentials. Otherwise, the test is skipped.
I pushed a commit with two changes. Feel free to revert if you don't like it.
- adding Twilio dependencies, otherwise the code doesn't compile.
- skipping the test if the Twilio credentials are not properly set.